packages/@okta/vuepress-site/docs/guides/oin-universal-logout-overview/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Build Universal Logout for your app
+meta:
+  - name: description
+    content: Provides an overview of Universal Logout and how to build a Universal Logout endpoint
+---
+
+## Universal Logout
+
+When an Identity Provider (IdP) like Okta detects identity threats or responds to employee termination events, it can prevent the user from signing in to apps in the future. However, this doesn't affect a user's existing sessions or tokens within an app.
```

Comment:
This is somewhat confusing to me, first time I read the first sentence I wasn't sure if you were describing the new logout api or not. Perhaps we can add *how* we prevent the user from the threat so it's clear this is not what the logout api will do.
```suggestion
When an Identity Provider (IdP) like Okta detects identity threats or responds to employee termination events, it can prevent the user from signing in to apps in the future by suspending, deactivating or deleting the user. However, this doesn't affect a user's existing sessions or tokens within an app.
```

Reply:
That sounds ok to me too - the important part was to understand this is fundamentally different than what we'll be describing later in this guide